src/components/skills-quiz/SearchCourseCard.jsx

```diff
@@ -36,8 +36,9 @@ const linkToCourse = (course, slug) => {
     queryId: course.queryId,
     objectId: course.objectId,
   };
+  const { userId } = getAuthenticatedUser();
   sendTrackEvent('edx.learner_portal.skills_quiz.course.clicked',
-    { userName: getAuthenticatedUser(), enterprise: slug, selectedCourse: course.key });
+    { user_id: userId, enterprise: slug, selectedCourse: course.key });
```

Comment:
would it be better to use camelCase here as well? i.e. `{ userId: userId, ... } `